src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -435,11 +427,15 @@ another.
     my_diffusion_pipeline/
         sub-01/
             dwi/
-                sub-01_dti.nii.gz
-                sub-01_dti.json
+                sub-01_desc-dti_parameter-all_model.nii.gz
+                sub-01_desc-dti_parameter-bzero_model.nii.gz
+                sub-01_desc-dti_parameter-fa_mdp.nii.gz
+                sub-01_desc-dti_model.json
     ```
 
-    Dimensions of NIfTI image "`sub-01_dti.nii.gz`": *I*x*J*x*K*x6 ([parameter vectors](#data-param))
+    Dimensions of NIfTI image "`sub-01_desc-dti_parameter-all_model.nii.gz`": *I*x*J*x*K*x6 ([parameter vectors](#data-param))
+    Dimensions of NIfTI image "`sub-01_desc-dti_parameter-bzero_model.nii.gz`": *I*x*J*x*K* ([scalar](#data-scalar))
```

Comment:
On the one hand, the first file among these two has an "all" in its file name suggesting that all relevant parameters are found in this file. On the other hand, we immediately have another `_model` file right next to it. That's a bit confusing, isn't it? So either the b0 goes into the `_all_model` file, or we need to find another way to find all the relevant model parameters distributed across multiple files.

Reply:
But otherwise, I think this is good to go.

Reply:
That's an error on my part. "`_param-all`" should only appear if it's the only image containing model parameters. If the tensor fitting also yields a *b*=0 estimate, it needs to be something like "`_param-tensor`" and "`_param-bzero`".

Reply:
Indeed exactly this was already described in the prior table. Should be resolved in https://github.com/bids-standard/bids-bep016/pull/46/commits/d3bbb8686a3b4c6298bcee29f4df51e199dfc720.